### PR TITLE
VR-150 VR Keyboard: Change Open/Close/Enter Behavior

### PR DESCRIPTION
**Pull request by SolarCrane**

### CODE REVIEW

File: CoreScriptsRoot/Modules/VR/VirtualKeyboard.lua [hunk 1]
```diff
@@ -847,6 +857,10 @@ local function ConstructKeyboardUI(keyboardLayoutDefinitions)
 	local textChangedConn = nil
 	local panelClosedConn = nil
 	rawset(newKeyboard, "Open", function(self, options)
+		if opened then return end
+		opened = true
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/VR/VirtualKeyboard.lua]: wat

File: CoreScriptsRoot/Modules/VR/Panel3D.lua [hunk 2]
```diff
@@ -781,7 +781,9 @@ local function onCameraChanged(prop)
 			currentHeadScale = workspace.CurrentCamera.HeadScale
 		end)
 		for i, v in pairs(panels) do
-			v:OnHeadScaleChanged(currentHeadScale)
```
Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/VR/Panel3D.lua]: You should discard lines 784-786, I just fixed this in PR 411 (https://github.com/ROBLOX/Core-Scripts/pull/411)

File: CoreScriptsRoot/Modules/VR/Panel3D.lua [hunk 3]
```diff
@@ -781,7 +781,9 @@ local function onCameraChanged(prop)
...
+			if v.OnHeadScaleChanged ~= nil then
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/VR/Panel3D.lua]: probably don't need this with kyle's change

Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/VR/Panel3D.lua]: ninja'd

**Pull request merged**